src/chains/celo/formatters.test.ts

```diff
@@ -862,5 +860,85 @@ describe('transactionRequest', () => {
         "value": "0x1",
       }
     `)
+
+    expect(
+      transactionRequest.format({
+        from: '0x0f16e9b0d03470827a95cdfd0cb8a8a3b46969b9',
+        to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8', // Recipient (illustrative address)
+        value: 1n,
+        feeCurrency: '0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1', // cUSD fee currency
+        maxFeePerGas: 2n, // Special field for dynamic fee transaction type (EIP-1559)
+        maxPriorityFeePerGas: 2n, // Special field for dynamic fee transaction type (EIP-1559)
+      }),
+    ).toMatchInlineSnapshot(`
+      {
+        "feeCurrency": "0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1",
+        "from": "0x0f16e9b0d03470827a95cdfd0cb8a8a3b46969b9",
+        "gas": undefined,
+        "gasPrice": undefined,
+        "maxFeePerGas": "0x2",
+        "maxPriorityFeePerGas": "0x2",
+        "nonce": undefined,
+        "to": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
+        "type": "0x7b",
+        "value": "0x1",
+      }
+    `)
+
+    expect(
```

Comment:
one suggestion is to write a comment saying this is testing cip42, and above the other cip64. etc